README.md

```diff
@@ -1,182 +1,177 @@
 
-#### Linux
+### As a CLI tool
 
-Meet the [node-gyp](https://github.com/nodejs/node-gyp#installation) requirements:
+You can alternatively use node-wot via its command line interface (CLI). Please visit the [CLI tool's Readme](<[url](https://github.com/eclipse-thingweb/node-wot/tree/master/packages/cli)>) to find out more.
 
--   Python v3.6, v3.7, or v3.8
--   make
--   A proper C/C++ compiler toolchain, like GCC
+#### As a docker image
 
-#### Windows
+Another option is to use node-wot inside a docker image. Make sure you are under Linux or under WSL if you are running on Windows.
 
-Install the Windows build tools through a CMD shell as administrator:
+Clone the repository:
 
 ```
-npm install -g --production windows-build-tools
+git clone https://github.com/eclipse-thingweb/node-wot
 ```
 
-> WSL: Windows Services for Linux should follow Linux instructions.
-
-#### Mac OS
-
-Meet the [node-gyp](https://github.com/nodejs/node-gyp#installation) requirements:
+Go into the repository:
 
 ```
-xcode-select --install
+cd node-wot
 ```
 
-### To use in a browser
-
-To use node-wot as a browser-side JavaScript Library, the browser needs to support ECMAScript 2015.
-Supported browsers include:
-
--   Microsoft Edge 15 and later
--   Firefox 54 and later
--   Chrome 58 and later
--   Safari 10 and later
-
-Using a browser with only ES5 support (eg. IE 11) might be possible if you add polyfills.
-
-## How to get the library
-
-### As a Node.js dependency
-
-You can install node-wot in the following ways:
-
-1. As a normal dependency (i.e., like loadsh). In this case, you are embedding a servient inside your application.
-2. As a CLI to run scripts. In this case, your application is running inside the default servient.
+Build the Docker image named `wot-servient` from the `Dockerfile`:
 
-#### Normal Dependency
+```
+npm run build:docker
+```
 
-If you want to use node-wot as a library in your Node.js application, you can use npm to install the node-wot packages that you need. To do so, `cd` inside your application folder, and run:
+Run the wot-servient as a container:
 
 ```
-npm i @node-wot/core @node-wot/binding-http --save
+docker run --rm wot-servient -h
 ```
 
-Now, you can implement a Thing as follows:
+## Examples
+
+With node-wot you can create server-side Things, in WoT jargon we call this operation "expose a thing" or you can create client-side Things, in WoT gergo we call this operation "consume a thing". An exposed Thing allows you to bring your device or services to the Web with just a few lines of code. On the other hand, with a consumed Thing, you have a fixed interface to interact with devices, potentially using different protocols/frameworks. In the following section, we will show how to create a simple counter Thing and how to consume it. Assuming you have installed and configured node-wot as a library, you can create and expose a counter Thing as follows:
```

Comment:
Just noticed another nit (although the Italian version also sounds great 🙂)

```suggestion
With node-wot you can create server-side Things, in WoT jargon we call this operation "expose a thing" or you can create client-side Things, in WoT jargon we call this operation "consume a thing". An exposed Thing allows you to bring your device or services to the Web with just a few lines of code. On the other hand, with a consumed Thing, you have a fixed interface to interact with devices, potentially using different protocols/frameworks. In the following section, we will show how to create a simple counter Thing and how to consume it. Assuming you have installed and configured node-wot as a library, you can create and expose a counter Thing as follows:
```

Reply:
ahahha I thought it was Latin or something :D :D committing. Thanks, Jan!